pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -67,6 +67,10 @@ python3Packages.buildPythonApplication rec {
     runHook postCheck
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/soundconverter --prefix PYTHONPATH : "$PYTHONPATH"
```

Comment:
https://ryantm.github.io/nixpkgs/languages-frameworks/gnome/#ssec-gnome-common-issues-double-wrapped
Does that binary get wrapped three times?

Reply:
Doesn't look like a wrapper:
```
 > file /nix/store/8j3w9jy82a6skdr0jhyzkrl2lx0h3wfw-soundconverter-4.0.4/bin/soundconverter    
/nix/store/8j3w9jy82a6skdr0jhyzkrl2lx0h3wfw-soundconverter-4.0.4/bin/soundconverter: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /nix/store/7jiqcrg061xi5clniy7z5pvkc4jiaqav-glibc-2.38-27/lib/ld-linux-x86-64.so.2, for GNU/Linux 3.10.0, not stripped
```
I guess it's a binary wrapper? How does one count number of wrappers?

Reply:
You can `cat` the binary file to see the hints (plain text), and then `ls -a` to see .wrapped files.

Reply:
Yes, it is

```
 ➜ tree ./result/bin/
soundconverter                             .soundconverter-wrapped_                   ...soundconverter-wrapped-wrapped-wrapped
.soundconverter-wrapped                    ..soundconverter-wrapped-wrapped
```

see eg https://github.com/NixOS/nixpkgs/blob/5822070afebef7cecd2e15312b158bff45816546/pkgs/applications/audio/mpdevil/default.nix#L38

Reply:
I see, thanks for explaining. But if I try to use this `makeWrapperArgs` method it does not work:
```
 > /nix/store/62h283p1bv87cp977q7y8sbg2cmds29q-soundconverter-4.0.4/bin/soundconverter 
/nix/store/62h283p1bv87cp977q7y8sbg2cmds29q-soundconverter-4.0.4/bin/..soundconverter-wrapped-wrapped:35: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
Traceback (most recent call last):
  File "/nix/store/62h283p1bv87cp977q7y8sbg2cmds29q-soundconverter-4.0.4/bin/..soundconverter-wrapped-wrapped", line 94, in <module>
    from soundconverter.interface.preferences import rates
ImportError: cannot import name 'rates' from 'soundconverter.interface.preferences' (/etc/profiles/per-user/jakubgs/lib/python3.11/site-packages/soundconverter/interface/preferences.py)
```
Not sure what's the correct way to fix this if I can't wrap it.

Reply:
Try replacing $PYTHONPATH with makePythonPath (eg https://github.com/NixOS/nixpkgs/pull/286846/files) or $out/${python.sitePackages}